packages/app/lib/updates.js

```diff
@@ -0,0 +1,10 @@
+const Update = require('../models/Update')
+
+exports.createNewUpdate = async update => {
+  const newUpdate = new Update({
+    ...update,
+    date: Date.now()
+  })
+  const savedUpdate = await newUpdate.save()
+  return savedUpdate
+}
```

Comment:
Can we have logging across every bit of the backend? I see you've added most places which is good.